### PR TITLE
KER-363

### PR DESCRIPTION
**Pull request by arpargo**
Upgrade all of the deprecated libraries.

### CODE REVIEW

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/createStore.js [hunk 1]
```diff
@@ -15,15 +14,16 @@ import rootReducer from './reducers';
 
 export const history = createBrowserHistory();
 
+
```
Comment (mikkojamG) [inline src/createStore.js:17]: Commented code should be removed 👀

File: src/views/Hearings/index.jsx [hunk 2]
```diff
@@ -390,24 +379,15 @@ Hearings.propTypes = {
...
   language: PropTypes.string,
-  location: PropTypes.shape({
-    search: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
-  }),
-  match: PropTypes.shape({
-    params: PropTypes.shape({
-      tab: PropTypes.string,
-    }),
-  }),
+  // location: PropTypes.shape({
```
Comment (mikkojamG) [inline src/views/Hearings/index.jsx:388]: Commented code should be removed 👀

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/components/Hearing/Comment/index.jsx [hunk 3]
```diff
@@ -12,71 +12,54 @@ import classnames from 'classnames';
...
-      if (toggleContainer) {
-        // finds the first anchor element and sets focus on it.
-        toggleContainer.querySelector('a').focus();
-      }
-    }
-  }
+  /**
+   * Determines whether the user type is admin.
+   */
+  const isAdminUser = () => typeof props.data.organization === 'string' || Array.isArray(props.data.organization);
```
Comment (mikkojamG) [inline src/components/Hearing/Comment/index.jsx:59]: Probably could just use isAdmin util function instead 🤔

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/components/admin/HearingFormStep1.jsx [hunk 4]
```diff
@@ -220,9 +217,39 @@ class HearingFormStep1 extends React.Component {
...
+                <ContactCard {...contact} />
+                <Button
+                  size='small'
+                  className='kerrokantasi-btn'
+                  onClick={() => this.openContactModal(contact)}
+                >
+                  <Icon className='icon' name='edit' />
+                </Button>
+              </div>);
+              /*
```
Comment (mikkojamG) [inline src/components/admin/HearingFormStep1.jsx:234]: Commented code should be removed 👀

**Review by mikkojamG — CHANGES_REQUESTED** [review]

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1038-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by mikkojamG — APPROVED** [review]
🎉

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1038-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**